src/Engine/ProtoCore/ProtoCore.csproj

```diff
@@ -30,11 +30,7 @@
     <PackageReference Include="System.Reflection.Emit" Version="4.7.0" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="FFI\FFICppFunction.cs" />
-    <None Include="FFI\PInvokeFFI.cs" />
-    <Compile Condition=" '$(TargetFramework)' == 'net6.0' " Remove="FFI\FFICppFunction.cs" />
-    <Compile Condition=" '$(TargetFramework)' == 'net6.0' " Remove="FFI\PInvokeFFI.cs" />
```

Comment:
Have you removed these files from the project altogether for both net48 and net6.0?

Reply:
no, the opposite, this code was **not** including these files for compilation, now I am including them.